### ISSUE TITLE
Fertilizer Hose Coupling

### ISSUE THREAD

**Issue by Bullitt33** [stijnwop/manureSystem#148]
Hello I am writing with an issue I am having trying to make a connector to hook 2 fertilizer hoses together to achieve a longer hose. Does anyone know of any such connector or know if there is any plans to have an updated that has the ability to hook more than 1 fertilizer hose together? Kind of frustrated not being able to get close enough with bigger equipment to connect the hose, but love the mod and do not want to stop using it either.

**Comment by Gordon-Dry** [comment]
There are two types of hoses, check the option in the shop.

**Comment by Bullitt33** [comment]
For the slurry hoses yes there are 2 types. But for liquid fertilizer there is only 1 option

**Comment by Gordon-Dry** [comment]
I see two types on both. Perhaps something is borked on your install.

**Comment by Bullitt33** [comment]
Can post a picture by chance?

**Comment by Gordon-Dry** [comment]
![fsScreen_2021_02_22_22_53_27](https://user-images.githubusercontent.com/29145213/108775288-2d774200-7561-11eb-8e90-9a664e03063c.png)
![fsScreen_2021_02_22_22_53_28](https://user-images.githubusercontent.com/29145213/108775301-2f410580-7561-11eb-932a-ddb814680223.png)
![fsScreen_2021_02_22_22_55_55](https://user-images.githubusercontent.com/29145213/108775303-30723280-7561-11eb-91a5-444ecc3ea6af.png)
![fsScreen_2021_02_22_22_55_57](https://user-images.githubusercontent.com/29145213/108775307-323bf600-7561-11eb-955c-9c2718cb14e7.png)

**Comment by Bullitt33** [comment]
Ok so I am using the latest version from the mod hub and I don’t have that option for the fert hoses wtf

**Comment by stijnwop** [comment]
I think @Gordon-Dry is using some kind of modified version?

The actual manure system only supports extensions for slurry hoses at this point.

**Comment by KITT3000** [comment]
These are hoses from the Hof Bergmann map, not from the MS mod.

**Comment by Bullitt33** [comment]
@KITT3000 do you know if the hoses work with MS? @stijnwop any plans to add this feature in an update? I think it’s much needed.

**Comment by KITT3000** [comment]
@Bullitt33 yes, it work

**Comment by Gordon-Dry** [comment]
@KITT3000 @Bullitt33 not from "Hof Bergmann Map", from the mod FS19_lsfmUniversalTankPack.
This works with manureSystem and does not need that map.

**Comment by KITT3000** [comment]
@Gordon-Dry right, I forgot that this is a separate mod (it is always included in the map pack)
